source/libraries/ContractDeployer.ts

```diff
@@ -57,7 +58,15 @@ Deploying to: ${networkConfiguration.networkName}
         await this.whitelistTradingContracts();
 
         if(this.configuration.createGenesisUniverse) {
+            if (!this.configuration.isProduction) {
+                this.initializeLegacyRep();
+            }
+
             this.universe = await this.createGenesisUniverse();
+
+            if (!this.configuration.isProduction) {
+                this.migrateFromlegacyRep();
```

Comment:
I'm not sure I understand how this is intended to work on production. I thought the idea was that holders of legacy REP would all be migrated to regular REP automatically.  How will this be done?

Reply:
In a non production environment there is only one Legacy REP holder, the deployer of the contracts. See the part right above thise where in a non production environment we mint 11 million Legacy REP for the default address

Reply:
Right, but isn't the intent to auto-migrate legacy REP users on production?  Will they have to call `migrateBalancesFromLegacyRep` and `migrateAllowancesFromLegacyRep` manually, or how will these be called on production?  Just trying to make sure I understand the intended behavior.

Reply:
These _won't_ be called in production. The reason we're calling them in dev is so that everything is ready to go for development and on testnets. In production we'll need to run a script that migrates balances and allowances before the system functions

Reply:
Ok, gotcha. That's the part I was trying to clarify.